packages/web-client/app/templates/card-pay/balances.hbs

```diff
@@ -1,3 +1,5 @@
+{{page-title "Card Balances" prepend=true}}
```

Comment:
according to [the documentation](https://github.com/ember-cli/ember-page-title#page-title-helper), `prepend` has this as a default, so might as well remove it here and elsewhere?

Reply:
Done!